### ISSUE TITLE
Add “fixed” variables

### ISSUE THREAD

**Issue by somerandomdev49** [enact-lang/enact#69]
I always wanted other languages to have it, but i have found 0 languages with this feature.
Currently, in enact there’s currently only `const`, 

but let me introduce you to  the `fixed` ( or just `fix` ).
Basically it behaves like const before #68. Example with painting: So when the painting is fixed, you just can’t move it or put it somewhere else (assign to it), but when it’s constant, it is completely static and unchangeable, so you can’t even paint on the painting.
The syntax is the same as with const:
```enact
fixed list = [1, 2, 3]
list = [ 69, 42 ] // error
list[0] = 3 // :D
```

(heh this is #69)

**Comment by Dandigit** [comment]
@somerandomdev49 I believe Java & friends implement something like this, with `final`? It's a nice idea, particularly for cases like your example above and when initializing structs.

Thank you for the proposal, consider it approved 😉. I'll leave this issue open until I implement it alongside #68.

**Comment by somerandomdev49** [comment]
:D (btw you can just use the old const implementation, just rename a bunch of stuff to fixed :) )

**Comment by Dandigit** [comment]
This will no longer be implemented, see #73.
The language is changing significantly, and I'm afraid this doesn't really fit with the new explicit referencing that I have in mind.
